### PR TITLE
Ben/docu 753

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

**Review by lucasdigital — APPROVED** [review]
Reads well.

File: docs/command-reference.md [hunk 1]
```diff
@@ -1065,53 +1345,110 @@ OPTIONS
...
- migration del: Delete a migration.
- migration exclusion add: Add an exclusion to a migration.
- migration exclusion del: Remove an exclusion from a migration.
- migration list: List running and active migrations.
- * migration new: Create a new migration.
- migration run: Start or resume a migration.
- migration show: Get migration details.
- migration stop: Abort a migration.
- status: Get migration status.
+      * migration add: Create a new migration.
```
Comment (lucasdigital) [inline docs/command-reference.md:2061]: Is this asterisk applicable? Looks out of place.

**Review by lucasdigital — CHANGES_REQUESTED** [review]
Just check the placement of the asterisk.  If right, this is good to go.

**Review by lucasdigital — CHANGES_REQUESTED** [review]
Hive, being a real noun should be capitalised, unless used in a small case-limited context (in code). "hive agent" should be "Hive agent". Or, do we have some overriding style rule?

File: docs/command-reference.md (hunk 1 above)
Comment (benradstone) [reply docs/command-reference.md:2061]: The asterisk is displayed when the command isn't available. This is a copy of the terminal after you've connected to LDM/HVM and ran the help command.

The migration add wouldn't be available until you had configured the storages.

**Comment by benradstone** [comment]
> Hive, being a real noun should be capitalised, unless used in a small case-limited context (in code). "hive agent" should be "Hive agent". Or, do we have some overriding style rule?

This is a tricky one, as we've coined the term "hive agent" for this product, but a "hive agent" will connect an "Apache Hive" metastore as well as other flavours of database (such as Azure SQL).

I've tried to be as consistent as possible when referring to the "hive agent" and use lowercase. When I'm referring to an Apache Hive metastore or just the product, I will use uppercase.

File: docs/command-reference.md (hunk 1 above)
Comment (lucasdigital) [reply docs/command-reference.md:2061]: Noted! It's good then.

**Comment by lucasdigital** [comment]
Thanks, Ben. Your reasoning is clear and consistency is more important, here. Go ahead and merge.

**Pull request merged**